internal/commands/commands.go

```diff
@@ -32,6 +32,13 @@ type PackClient interface {
 	InspectExtension(client.InspectExtensionOptions) (*client.ExtensionInfo, error)
 	PullBuildpack(context.Context, client.PullBuildpackOptions) error
 	DownloadSBOM(name string, options client.DownloadSBOMOptions) error
+	CreateManifest(ctx context.Context, opts client.CreateManifestOptions) error
+	AnnotateManifest(ctx context.Context, opts client.ManifestAnnotateOptions) error
+	AddManifest(ctx context.Context, opts client.ManifestAddOptions) error
+	DeleteManifest(ctx context.Context, name []string) []error
+	RemoveManifest(ctx context.Context, name string, images []string) []error
```

Comment:
To keep the standard use in the other commands, we can create a `client.DeleteManifest` and `client.RemoveManifest` structures. I didn't do it because I wanted to un-draft the PR as soon as possible.